prac_08/taxi_simulator.py

```diff
@@ -0,0 +1,57 @@
+"""
+CP1404/CP5632 Practical
+Taxi Simulator
+"""
+from prac_08.taxi import Taxi
+from prac_08.silver_service_taxi import SilverServiceTaxi
+
+MENU = "q)uit, c)hoose taxi, d)rive"
+
+
+def main():
+    """Taxi Program."""
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
+    bill_to_date = 0.00
+
+    print("Let's drive!")
+    print(MENU)
+    choice = input(">>> ").lower()
+    while choice != "q":
+        if choice == "c":
+            print("Taxis available:")
+            display_available_taxis(taxis)
+            taxi_choice = int(input("Choose taxi: "))
```

Comment:
On both line 24 and 35 assume that the user will enter an integer, but you could also handle an incorrect type here to prevent a crash